benchmarks/models.py

```diff
@@ -19,6 +19,10 @@
 
 class ManifestReduced(models.Model):
     hash = models.CharField(primary_key=True, max_length=40)
+    created_at = models.DateTimeField(default=timezone.now)
```

Comment:
ah sorry I forgot to mention that I deleted the old data manually on my local dump, and I also already did that in production. you have to delete the manifests that have no associated builds first, then the reduced manifests that have no associated manifests later.

```sql
delete from benchmarks_manifest
where not exists
  (select 1 from benchmarks_result where manifest_id = benchmarks_manifest.id);
delete from benchmarks_manifestreduced
where not exists
  (select 1 from benchmarks_manifest where reduced_id = benchmarks_manifestreduced.hash);
```